Makefile

```diff
@@ -40,6 +40,15 @@ run : habitat c-backend c-frontend
 
 run-docker: habitat-docker c-backend-docker c-frontend
 
+test::
+	go test ./...
+
+test::
+	prove -v -r
```

Comment:
what does this do?

Reply:
prove runs all tap tests (test files are by convention in folders that are named `t`)